### PR TITLE
feat: link curation component refactor

### PR DESCRIPTION
**Pull request by hlibmykhailov**

### CODE REVIEW

File: src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js [hunk 1]
```diff
@@ -3,208 +3,181 @@ import * as typography from 'components/mixins/typography';
...
-              line-height: 2;
-            }
-          }
-        }
-      `};
-
-      @media (min-width: 580px) and (max-width: 1000px) {
-        max-width: calc(100% - 8px);
+    @media (min-width: 580px) and (max-width: 1000px) {
+      max-width: calc(100% - 8px);
```
Comment (oleksandrbulakh) [inline src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js]: when it's necessarily?

You have same prop at line 26.

File: src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js [hunk 2]
```diff
@@ -3,208 +3,181 @@ import * as typography from 'components/mixins/typography';
...
+    height: 195px;
+    box-shadow: 0 2px 10px 0 ${transparentize(defaultBoxShadowColor, 0.1)};
+    background-color: ${props => props.theme.colors.contentBodyColor};
+    box-sizing: border-box;
+    margin: 4px 4px 6px;
+    text-decoration: none;
+
+    ${media.mobile`
         flex-direction: column;
         max-width: calc(100% - 8px);
```
Comment (oleksandrbulakh) [inline src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js]: when it's necessarily?

You have same prop at line 26.

File: src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js [hunk 3]
```diff
@@ -3,208 +3,181 @@ import * as typography from 'components/mixins/typography';
...
+
+export const CustomDescription = styled.div``;
+
+export const ImageWrapper = styled.div`
+  display: flex;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+
+  &.parsed {
```
Comment (oleksandrbulakh) [inline src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js]: Where does this class set?

File: src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js [hunk 4]
```diff
@@ -3,208 +3,181 @@ import * as typography from 'components/mixins/typography';
...
-            }
-          }
-        }
-      }
+  &.deleted {
+    padding-left: 20px;
+  }
+
+  ${media.mobile`
+    &.parsed {
```
Comment (oleksandrbulakh) [inline src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js]: Where does this class set?

File: src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js [hunk 5]
```diff
@@ -3,208 +3,181 @@ import * as typography from 'components/mixins/typography';
...
-          .text-block {
-            .link-content {
-              max-height: 90px;
-              line-height: 2.14;
-              margin-bottom: 8px;
-            }
-          }
-        }
-      }
+  &.deleted {
```
Comment (oleksandrbulakh) [inline src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js]: Where does this class set?

File: src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js [hunk 6]
```diff
@@ -3,208 +3,181 @@ import * as typography from 'components/mixins/typography';
...
+    padding-left: 20px;
+  }
+
+  ${media.mobile`
+    &.parsed {
+      margin: 20px;
+    }
+
+    &.deleted {
```
Comment (oleksandrbulakh) [inline src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js]: Where does this class set?

File: src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js (hunk 1 above)
Comment (hlibmykhailov) [reply src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js]: Thanks, its was unnecessary

File: src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js (hunk 2 above)
Comment (hlibmykhailov) [reply src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js]: Thanks, its was unnecessary

File: src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js (hunk 3 above)
Comment (hlibmykhailov) [reply src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js]: Classes .parsed and .deleted set when image is parsed and its present or deleted for different styles

File: src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js (hunk 5 above)
Comment (hlibmykhailov) [reply src/components/contents/ContentItem/LinkCuration/LinkCuration.styled.js]: Classes .parsed and .deleted set when image is parsed and its present or deleted for different styles

**Pull request merged**